lua/nvim-treesitter/install.lua

```diff
@@ -31,7 +31,7 @@ local M = {}
 ---@type table<string, LockfileInfo>
 local lockfile = {}
 
-local max_jobs = 50
+local max_jobs = 10
```

Comment:
50 was way too much once `tree-sitter` got into the mix; the memory pressure skyrocketed. This is also friendlier for CI (macos went from 1:40 to 1:10 for compiling all parsers). @lewis6991